src/ring_theory/ideal/quotient_operations.lean

```diff
@@ -126,6 +126,10 @@ lemma mem_quotient_iff_mem {I J : ideal R} (hIJ : I ≤ J) {x : R} :
   quotient.mk I x ∈ J.map (quotient.mk I) ↔ x ∈ J :=
 by rw [mem_quotient_iff_mem_sup, sup_eq_left.mpr hIJ]
 
+lemma comap_quo_map_quo {I J : ideal R} (h : I ≤ J) :
```

Comment:
I don't think we use `quo` elsewhere, let's try to match existing names.

Reply:
Yes, I renamed it to `comap_quot_map_quot`. I am not sure it's the best name though.